recipes/rgbcx/all/conanfile.py

```diff
@@ -0,0 +1,44 @@
+import os
+import glob
+from conans import ConanFile, tools
+
+
+class RgbcxConan(ConanFile):
+    name = "rgbcx"
+    description = "High-performance scalar BC1-5 encoders."
+    homepage = "https://github.com/richgel999/bc7enc/blob/master/rgbcx.h"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("conan", "BC1", "BC5", "BCx", "encoding")
+    license = "MIT", "Unlicense"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = glob.glob('bc7enc-*/')[0]
+        os.rename(extracted_dir, self._source_subfolder)
+        self._patch_sources()
+
+    def _patch_sources(self):
+        tools.replace_in_file(os.path.join(self._source_subfolder, "rgbcx.h"),
+            "#include <stdlib.h>", "#include <stdlib.h>\n#include <string.h>")
+
+    def _extract_licenses(self):
+        header = tools.load(os.path.join(self.source_folder, self._source_subfolder, "rgbcx.h"))
+        mit_content = header[header.find("ALTERNATIVE A - "):header.find("ALTERNATIVE B -")]
+        tools.save("LICENSE_MIT", mit_content)
+        unlicense_content = header[header.find("ALTERNATIVE B - "):header.rfind("*/", 1)]
+        tools.save("LICENSE_UNLICENSE", unlicense_content)
+
```

Comment:
```suggestion
```

No required, only copy the original license.

Reply:
Original license also contains lodepng license.

Reply:
Also, the original license covers bc7enc. This recipe extracts rgbcx so i think it's better to extract the license from the header file.

Reply:
The license file already explains it, keep the original license file. If you think it should be separated, ask the author for splinting the license file instead.